README.md

```diff
@@ -1 +1,40 @@
-# hello-world
\ No newline at end of file
+# hello_world
+This repository is about creating a header in Html and giving it a color with a color property in CSS
+
+## Built with
+-HTML
+-CSS
+
+>To get a local copy running,follow these steps
+
+## Prerequisites
+-code Editor(preferrably Visual studio code editor)
+-Git
+-Github Account
+
+## setup to access the project
+(go to the terminal in the code editor and put the following git commands)
+-git init
+-git clone https://github.com/yusufnviiri/hello-world.git
+-cd hello-world
+-open index.html in the editor/browser to view the project contents
+-open style.css to see the CSS styles used in the project.
+-enjoy
+
+## Author
+
+### :MAN: Yusuf Nviiri
+
+-GitHub: @yusufnviiri
+-Twitter:@YNviiri
+-LinkedIn: yusuf nviiri
```

Comment:
- [ ] Kindly update all social media links, this helps people connect with you on other platforms 🤗

Reply:
thanks for the feedback